src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingOrderControllerTest.java

```diff
@@ -27,7 +36,25 @@
     public void whenStoreHearingOrder_expectedServicesAreInvoked() {
         hearingOrderController.storeHearingOrder(AUTH_TOKEN, buildCallbackRequest());
 
-        verify(hearingOrderService, times(1)).convertToPdfAndStampAndStoreLatestDraftHearingOrder(any(), eq(AUTH_TOKEN));
-        verify(caseDataService, times(1)).moveCollection(any(), eq(DRAFT_DIRECTION_DETAILS_COLLECTION), eq(DRAFT_DIRECTION_DETAILS_COLLECTION_RO));
+        verify(hearingOrderService).convertToPdfAndStampAndStoreLatestDraftHearingOrder(any(), eq(AUTH_TOKEN));
+        verify(caseDataService).moveCollection(any(), eq(DRAFT_DIRECTION_DETAILS_COLLECTION), eq(DRAFT_DIRECTION_DETAILS_COLLECTION_RO));
+    }
+
+    @Test
+    public void givenDraftDirectionOrderCollectionIsNotEmpty_whenStartingHearingOrderApproval_thenLatestDraftDirOrderIsPopulated() {
+        when(hearingOrderService.draftDirectionOrderCollectionTail(any())).thenReturn(Optional.of(DraftDirectionOrder.builder().build()));
+
+        ResponseEntity<AboutToStartOrSubmitCallbackResponse> response = hearingOrderController.startHearingOrderApproval(buildCallbackRequest());
+
+        assertThat(response.getBody().getData().get(LATEST_DRAFT_DIRECTION_ORDER), is(notNullValue()));
```

Comment:
can you validate that the value is equal to the return value from the mock above